data/41-nitrokey.rules

```diff
@@ -1,5 +1,7 @@
 # Nitrokey U2F
 KERNEL=="hidraw*", SUBSYSTEM=="hidraw", MODE="0664", GROUP="plugdev", ATTRS{idVendor}=="2581", ATTRS{idProduct}=="f1d0"
+# Nitrokey FIDO U2F
+KERNEL=="hidraw*", SUBSYSTEM=="hidraw", ATTRS{idVendor}=="20a0", ATTRS{idProduct}=="4287", TAG+="uaccess"
```

Comment:
Please check, should we not use `SUBSYSTEM=="usb"`, instead of `hidraw`.

Reply:
Why do you think we should? We did not use usb subsystem in any other rule. Furthermore, no [other U2F device](https://github.com/Yubico/libu2f-host/blob/master/70-u2f.rules) seems to use it.

Reply:
I am sorry - I do not remember anymore the reason. I guess I thought about greater generalization for the rule. Please skip this for now.